{{cookiecutter.directory_name}}/project_setup.md

```diff
@@ -42,7 +42,7 @@ help you decide which tool to use for packaging.
 - The `tests` folder contains:
   - Example tests that you should replace with your own meaningful tests (file: `test_my_module.py`)
 - The testing framework used is [PyTest](https://pytest.org)
-  - [PyTest introduction](http://pythontesting.net/framework/pytest/pytest-introduction/)
+  - [PyTest introduction](https://pythontest.com/pytest-book/)
```

Comment:
Sad that the book is behind a medium login wall, but at least a working link is better than a broken one.

Reply:
Yep. I didn't have time to look for a more accessible good source. This was really already outside of the scope of this PR, just wanted to fix it quickly.